ansible/site.yml

```diff
@@ -194,6 +201,8 @@
           parameters:
             key: "pvwproxy.proxy_file_path"
             value: "/opt/hpccloud/proxy"
+      when: not cumulus
```

Comment:
The particular order is required, so not sure we ca extract them out into another file.